examples/directml/llama_v2/config_llama_v2.json

```diff
@@ -13,207 +13,207 @@
-                                "cache_out.28.value",
-                                "cache_out.29.key",
-                                "cache_out.29.value",
-                                "cache_out.30.key",
-                                "cache_out.30.value",
-                                "cache_out.31.key",
-                                "cache_out.31.value"
+                                "past_key_values_out.0.key",
+                                "past_key_values_out.0.value",
+                                "past_key_values_out.1.key",
+                                "past_key_values_out.1.value",
+                                "past_key_values_out.2.key",
+                                "past_key_values_out.2.value",
+                                "past_key_values_out.3.key",
+                                "past_key_values_out.3.value",
+                                "past_key_values_out.4.key",
+                                "past_key_values_out.4.value",
+                                "past_key_values_out.5.key",
+                                "past_key_values_out.5.value",
+                                "past_key_values_out.6.key",
+                                "past_key_values_out.6.value",
+                                "past_key_values_out.7.key",
+                                "past_key_values_out.7.value",
+                                "past_key_values_out.8.key",
+                                "past_key_values_out.8.value",
+                                "past_key_values_out.9.key",
+                                "past_key_values_out.9.value",
+                                "past_key_values_out.10.key",
+                                "past_key_values_out.10.value",
+                                "past_key_values_out.11.key",
+                                "past_key_values_out.11.value",
+                                "past_key_values_out.12.key",
+                                "past_key_values_out.12.value",
+                                "past_key_values_out.13.key",
+                                "past_key_values_out.13.value",
+                                "past_key_values_out.14.key",
+                                "past_key_values_out.14.value",
+                                "past_key_values_out.15.key",
+                                "past_key_values_out.15.value",
+                                "past_key_values_out.16.key",
+                                "past_key_values_out.16.value",
+                                "past_key_values_out.17.key",
+                                "past_key_values_out.17.value",
+                                "past_key_values_out.18.key",
+                                "past_key_values_out.18.value",
+                                "past_key_values_out.19.key",
+                                "past_key_values_out.19.value",
+                                "past_key_values_out.20.key",
```

Comment:
instead of changing all those stuff, is it better to using a custom function like https://github.com/microsoft/Olive/blob/main/examples/llama2/user_script.py#L153 or https://github.com/microsoft/Olive/blob/461419f3fa4f5802914a49b8786430e9c5d85772/examples/phi2/phi2_optimize.json#L7 to avoid manually changing it whenever there are ay schema updates in future?

Reply:
just some two cents. You can decide whether do the change or not.

Reply:
This whole example will be replaced by another one soon-ish, but I'll keep this in mind for the other example.